### ISSUE TITLE
Extensions filter for publisher should be exact

### ISSUE THREAD

**Issue by aquarion** [microsoft/vsmarketplace#328]

**Comment by sandy081** [comment]
CC @isidorn

**Comment by isidorn** [comment]
I agree this is a bug, and we should fix this. For reference here's the image. 

![Screenshot 2022-02-09 at 11 47 20](https://user-images.githubusercontent.com/1926584/153182841-284a02bb-9eab-41e2-a282-2fd69c61b0c0.png)

**Comment by prashantvc** [comment]
We are tracking all search related issue at #154

**Comment by SaiKanth007** [comment]
Similar to categories filter, marketplace also provides support for publisherName filter, which can be leveraged to search for extensions belonging to a specific publisher.

@sandy081  VSCode can map to this filter for filtering on publisherName (filterType=18)
